.azure/pipelines/ci.yml

```diff
@@ -52,6 +57,10 @@ variables:
   - name: _InternalRuntimeDownloadCodeSignArgs
     value: /p:DotNetRuntimeSourceFeed=https://dotnetclimsrc.blob.core.windows.net/dotnet /p:DotNetRuntimeSourceFeedKey=$(dotnetclimsrc-read-sas-token-base64)
   - ${{ if notin(variables['Build.Reason'], 'PullRequest') }}:
+    - name: WindowsArm64LogArgs
+      value: -ExcludeCIBinaryLog
+    - name: WindowsArm64InstallersLogArgs
+      value: -ExcludeCIBinaryLog
```

Comment:
minor but binary log creation isn't conditional in this branch. Might as well add the `/bl:...` arguments to the build steps directly